src/components/pollList.vue

```diff
@@ -1,12 +1,20 @@
 <template>
-  <div class="home">
+  <div class="pollHome">
     <div class="pollList">
       <div class="poll" v-for="poll in polls" :key="poll.id">
-        <h3 @click="showPoll(polls.indexOf(poll))">{{ poll.title }}</h3>
+        <h3>
```

Comment:
HTML structure is not good, it should be in a div, heading, para tags should not contain child elements @PiyushInnotech 
change the HTML structure.